utils/java/src/main/java/com/logicalclocks/utils/MainClass.java

```diff
@@ -130,6 +132,15 @@ public static void main(String[] args) throws Exception {
       }
       if (!success) {
         System.exit(1);
```

Comment:
Won't the System.exit(1); on line 134 prevent the added code below from running?

Reply:
Sorry this was my mistake. There was a duplication of code I fixed it now.